src/pages/participant/ParticipantPage.tsx

```diff
@@ -253,6 +254,15 @@ function ParticipantPageContent({
             />
             <div className="flex flex-wrap items-center justify-center gap-2">
               <RegistrationState state={participant.state} />
+              {participant.tags
+                .sort((a, b) =>
+                  (typeof a === 'string' ? a : a.title).localeCompare(
+                    typeof b === 'string' ? b : b.title
```

Comment:
I would normalize the tags when loading the data, not when showing them

Reply:
ie right after receiving data from the server - before putting it in the local database - convert string tags to tag objects (w/ whatever default color those get)

Reply:
fixed it on the server side, was using the wrong marshmallow field :)